2021-07-05 00:00:00,093 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:03,096 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:06,116 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:09,135 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:12,140 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:15,148 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:18,154 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:21,167 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:24,184 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:27,190 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:30,208 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:33,227 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:36,235 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:39,245 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:42,259 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:45,261 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:48,268 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:51,286 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:54,304 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:00:57,326 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:00,328 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:03,335 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:06,351 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:09,365 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:12,381 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:15,400 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:18,413 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:21,417 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:24,424 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:27,444 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:30,463 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:33,464 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:36,471 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:39,494 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:42,501 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:45,521 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:48,542 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:51,546 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:54,554 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:01:57,571 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:00,590 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:03,606 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:06,618 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:09,625 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:12,631 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:15,650 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:18,657 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:21,654 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:24,661 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:27,681 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:30,687 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:33,688 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:36,690 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:39,709 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:42,711 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:45,715 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:48,722 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:51,745 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:54,748 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:02:57,755 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:00,772 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:03,785 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:06,803 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:09,825 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:12,829 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:15,852 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:18,865 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:21,876 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:24,876 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:27,889 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:30,894 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:33,915 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:36,929 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:39,948 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:42,948 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:45,955 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:48,965 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:51,962 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:54,981 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:03:57,981 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:00,989 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:04,006 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:07,015 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:10,021 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:13,028 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:16,037 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:19,061 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:22,065 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:25,088 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:28,096 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:31,122 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:34,135 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:37,151 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:40,167 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:43,175 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:46,201 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:49,202 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:52,205 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:55,230 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:04:58,232 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:01,241 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:04,252 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:07,269 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:10,285 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:13,304 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:16,305 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:19,308 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:22,316 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:25,335 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:28,357 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:31,368 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:34,375 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:37,379 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:40,401 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:43,401 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:46,407 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:49,412 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:52,434 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:55,439 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:05:58,465 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:01,474 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:04,492 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:07,498 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:10,501 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:13,524 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:16,529 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:19,530 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:22,541 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:25,548 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:28,568 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:31,575 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:34,576 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:37,584 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:40,594 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:43,599 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:46,622 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:49,629 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:52,644 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:55,663 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:06:58,673 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:01,699 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:04,707 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:07,720 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:10,728 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:13,722 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:16,743 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:19,748 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:22,757 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:25,765 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:28,770 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:31,796 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:34,806 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:37,819 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:40,826 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:43,830 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:46,848 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:49,875 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:52,879 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:55,900 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:07:58,915 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:01,933 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:04,937 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:07,945 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:10,956 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:13,973 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:16,985 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:19,977 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:22,983 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:25,982 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:28,990 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:31,999 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:35,005 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:38,028 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:41,031 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:44,041 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:47,048 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:50,067 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:53,086 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:56,102 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:08:59,108 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:02,106 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:05,114 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:08,120 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:11,126 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:14,133 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:17,150 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:20,170 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:23,177 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:26,183 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:29,195 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:32,216 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:35,220 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:38,236 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:41,255 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:44,270 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:47,296 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:50,304 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:53,309 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:56,316 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:09:59,328 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:02,338 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:05,342 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:08,335 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:11,358 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:14,368 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:17,370 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:20,377 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:23,395 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:26,405 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:29,414 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:32,422 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:35,424 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:38,445 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:41,452 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:44,450 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:47,460 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:50,469 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:53,474 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:56,483 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:10:59,492 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:02,510 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:05,515 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:08,521 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:11,531 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:14,538 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:17,546 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:20,558 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:23,566 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:26,590 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:29,593 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:32,604 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:35,609 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:38,612 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:41,635 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:44,642 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:47,670 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:50,671 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:53,677 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:56,683 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:11:59,693 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:02,701 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:05,706 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:08,719 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:11,728 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:14,735 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:17,756 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:20,773 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:23,777 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:26,780 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:29,790 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:32,799 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:35,820 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:38,824 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:41,828 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:44,835 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:47,844 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:50,850 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:53,861 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:56,881 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:12:59,905 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:02,910 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:05,928 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:08,950 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:11,959 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:14,981 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:17,987 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:21,011 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:24,028 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:27,042 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:30,043 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:33,056 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:36,061 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:39,064 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:42,088 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:45,098 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:48,125 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:51,126 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:54,147 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:13:57,169 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:00,171 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:03,182 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:06,187 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:09,192 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:12,206 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:15,215 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:18,222 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:21,239 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:24,264 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:27,264 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:30,272 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:33,274 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:36,293 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:39,294 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:42,301 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:45,324 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:48,329 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:51,337 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:54,347 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:14:57,364 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:00,384 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:03,389 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:06,394 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:09,403 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:12,413 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:15,432 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:18,438 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:21,465 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:24,481 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:27,488 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:30,505 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:33,524 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:36,534 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:39,555 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:42,558 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:45,568 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:48,588 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:51,593 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:54,604 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:15:57,612 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:00,632 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:03,637 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:06,643 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:09,667 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:12,679 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:15,691 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:18,714 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:21,724 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:24,741 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:27,764 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:30,774 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:33,779 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:36,778 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:39,779 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:42,789 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:45,797 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:48,805 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:51,826 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:54,829 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:16:57,856 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:00,859 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:03,868 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:06,875 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:09,881 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:12,893 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:15,912 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:18,923 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:21,944 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:24,952 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:27,957 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:30,964 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:33,978 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:36,987 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:40,012 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:43,016 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:46,036 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:49,047 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:52,053 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:55,062 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:17:58,070 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:01,093 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:04,106 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:07,107 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:10,114 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:13,125 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:16,131 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:19,150 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:22,157 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:25,184 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:28,191 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:31,214 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:34,213 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:37,225 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:40,236 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:43,261 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:46,266 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:49,272 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:52,286 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:55,306 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:18:58,319 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:01,341 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:04,345 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:07,369 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:10,378 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:13,396 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:16,418 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:19,422 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:22,427 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:25,442 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:28,461 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:31,464 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:34,464 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:37,465 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:40,481 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:43,489 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:46,511 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:49,522 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:52,540 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:55,550 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:19:58,546 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:01,555 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:04,566 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:07,584 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:10,592 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:13,597 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:16,619 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:19,628 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:22,652 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:25,659 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:28,687 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:31,689 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:34,712 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:37,719 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:40,724 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:43,749 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:46,753 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:49,770 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:52,788 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:55,796 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:20:58,804 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:01,812 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:04,826 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:07,840 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:10,844 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:13,853 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:16,873 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:19,884 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:22,891 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:25,898 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:28,904 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:31,913 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:34,914 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:37,922 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:40,932 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:43,947 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:46,954 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:49,961 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:52,957 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:55,974 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:21:58,983 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:01,983 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:04,988 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:08,014 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:11,020 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:14,040 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:17,045 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:20,052 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:23,054 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:26,060 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:29,068 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:32,081 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:35,092 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:38,099 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:41,117 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:44,141 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:47,151 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:50,153 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:53,158 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:56,182 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:22:59,189 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:02,206 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:05,228 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:08,236 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:11,256 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:14,267 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:17,290 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:20,302 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:23,325 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:26,329 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:29,333 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:32,351 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:35,354 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:38,366 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:41,376 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:44,402 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:47,408 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:50,423 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:53,427 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:56,447 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:23:59,455 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:02,477 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:05,480 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:08,479 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:11,493 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:14,500 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:17,502 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:20,521 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:23,538 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:26,551 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:29,571 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:32,587 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:35,594 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:38,607 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:41,609 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:44,612 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:47,634 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:50,644 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:53,657 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:56,678 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:24:59,697 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:02,705 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:05,727 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:08,740 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:11,751 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:14,758 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:17,760 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:20,764 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:23,770 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:24,893 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\ConnBase.py:192"]: peer addr: ('127.0.0.1', 9100), close_reason='[WinError 64] The specified network name is no longer available'
2021-07-05 00:25:26,774 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:40,621 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:25:40,623 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:25:40,627 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe1\x04\xec\x85\xf8\x0e\xc3\xc9\xe8\xe1' RpcHandler Destroy!
2021-07-05 00:25:40,628 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_0', '127.0.0.1', 9201)
2021-07-05 00:25:40,630 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e104ec85f80ec3c9e8e2 mem_id=2321150570944
2021-07-05 00:25:40,631 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e104ec85f80ec3c9e8e2 
2021-07-05 00:25:40,636 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9201) is connected !!!!
2021-07-05 00:25:40,669 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:40,701 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:43,682 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:43,692 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:46,688 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:46,700 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:49,684 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:49,694 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:52,693 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:52,703 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:55,698 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:55,710 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:58,700 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:25:58,711 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9201)
2021-07-05 00:26:00,007 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\ConnBase.py:192"]: peer addr: ('127.0.0.1', 9100), close_reason='[WinError 64] The specified network name is no longer available'
2021-07-05 00:28:33,476 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:28:34,083 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:28:34,430 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:28:36,138 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:28:36,727 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 2
2021-07-05 00:28:37,440 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:28:38,779 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:28:39,387 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 3
2021-07-05 00:28:40,459 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:28:41,429 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:28:42,036 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 4
2021-07-05 00:28:43,472 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:28:44,092 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:28:44,699 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 5
2021-07-05 00:29:31,867 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:29:33,896 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:29:34,503 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 00:29:34,878 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:30:19,196 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:30:19,201 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:30:23,778 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 2
2021-07-05 00:30:23,783 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:30:25,820 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:30:26,440 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 3
2021-07-05 00:30:26,797 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:30:31,641 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:30:32,245 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:30:32,572 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:30:34,302 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:30:34,910 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 2
2021-07-05 00:30:35,597 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:31:06,599 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:31:06,639 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:31:11,268 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:31:12,548 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe2J\x08W*\xb1\x80\xa5\x1b\x05' RpcHandler Destroy!
2021-07-05 00:31:12,551 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:31:12,555 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e25008572ab180a51b06 mem_id=2133163195312
2021-07-05 00:31:12,559 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e25008572ab180a51b06 
2021-07-05 00:31:12,569 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:31:12,608 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:31:16,392 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:31:16,398 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:31:16,409 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:31:19,391 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:31:19,405 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:31:22,222 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:31:22,266 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:31:22,270 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:31:22,275 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe2Z]\xed9\xd7\xb8\xd9\xd9\xdb' RpcHandler Destroy!
2021-07-05 00:31:22,277 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:31:22,281 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e25a5ded39d7b8d9d9dc mem_id=1885071850560
2021-07-05 00:31:22,284 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e25a5ded39d7b8d9d9dc 
2021-07-05 00:31:22,293 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:31:22,331 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:31:22,351 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:31:26,741 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 00:31:28,818 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:31:39,079 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:31:41,171 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:31:41,782 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:31:42,079 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:31:42,086 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:31:43,835 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:31:44,443 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 2
2021-07-05 00:31:45,095 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:31:45,102 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:31:46,475 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:31:47,085 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 3
2021-07-05 00:31:48,113 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:31:48,120 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:31:49,139 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:31:49,745 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 4
2021-07-05 00:31:51,128 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:31:51,135 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:31:51,798 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:31:52,406 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 5
2021-07-05 00:31:54,134 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:31:54,141 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:31:54,444 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:31:55,054 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 6
2021-07-05 00:31:55,057 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:65"]: try 6 times , still can't connect tcp remote addr: ('127.0.0.1', 9102)
2021-07-05 00:31:57,173 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:32:00,201 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:34:03,348 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:34:03,395 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:34:05,438 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:34:06,045 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:34:06,406 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:35:29,003 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:35:31,082 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:35:31,690 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:35:32,018 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:35:32,021 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:35:32,024 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:35:32,027 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:35:32,028 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:35:32,036 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe3P\x9e\xac\x95B\xc8\xd7\x13,' RpcHandler Destroy!
2021-07-05 00:35:32,037 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:35:32,039 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e3549eac9542c8d7132d mem_id=1918157732064
2021-07-05 00:35:32,041 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e3549eac9542c8d7132d 
2021-07-05 00:35:32,045 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:35:32,081 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:32,103 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:35,098 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:35,110 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:38,102 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:38,113 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:40,346 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:35:42,425 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:35:43,032 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:35:43,363 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:35:45,086 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:35:45,694 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 2
2021-07-05 00:35:46,379 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:35:46,382 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:35:46,386 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:35:46,389 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:35:46,390 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:35:46,392 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:35:46,401 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe3\\\xc3\x81\\\x8b|\xaeL\xf1' RpcHandler Destroy!
2021-07-05 00:35:46,402 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:35:46,404 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e362c3815c8b7cae4cf2 mem_id=2693057614416
2021-07-05 00:35:46,406 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e362c3815c8b7cae4cf2 
2021-07-05 00:35:46,411 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:35:46,441 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:46,461 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:49,461 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:49,473 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:52,466 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:52,477 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:55,469 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:55,480 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:58,476 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:35:58,486 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:01,481 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:01,492 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:04,482 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:04,494 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:07,496 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:07,506 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:10,509 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:10,520 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:13,516 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:13,526 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:16,529 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:16,539 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:19,540 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:19,550 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:22,548 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:22,558 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:25,558 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:25,570 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:28,556 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:28,567 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:31,560 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:31,569 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:34,564 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:34,574 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:37,583 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:37,593 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:40,594 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:40,605 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:43,598 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:43,614 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:46,613 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:46,625 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:49,617 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:49,627 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:52,624 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:52,634 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:55,629 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:55,639 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:58,646 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:36:58,657 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:01,656 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:01,667 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:04,658 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:04,668 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:07,673 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:07,684 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:10,677 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:10,687 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:13,677 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:13,687 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:16,687 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:16,697 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:19,691 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:19,703 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:22,714 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:22,724 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:25,724 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:25,734 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:28,726 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:28,736 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:31,743 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:31,752 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:34,749 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:34,768 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:37,750 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:37,761 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:40,753 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:40,763 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:43,775 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:43,784 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:46,792 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:46,805 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:49,808 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:49,818 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:52,822 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:52,833 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:55,841 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:55,850 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:58,842 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:37:58,852 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:01,849 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:01,858 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:04,867 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:04,878 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:07,867 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:07,876 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:10,886 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:10,898 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:13,886 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:13,898 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:16,888 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:16,899 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:19,903 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:19,915 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:22,912 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:22,922 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:25,915 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:25,925 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:28,914 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:28,925 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:31,928 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:31,939 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:34,945 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:38:34,955 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:39:12,924 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:39:53,394 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:40:04,252 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:40:04,258 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:40:20,754 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 00:40:29,818 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:40:31,660 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:40:39,035 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:40:40,807 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:40:40,814 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 00:40:48,941 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:40:48,948 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 2
2021-07-05 00:41:03,977 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:41:04,857 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:41:04,863 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 2
2021-07-05 00:41:06,121 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:41:06,127 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 3
2021-07-05 00:41:13,208 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:41:13,707 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:41:13,712 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:41:22,367 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:41:32,261 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:41:32,264 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 3
2021-07-05 00:41:33,143 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 4
2021-07-05 00:42:57,380 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:42:58,964 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:43:00,458 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:43:01,903 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:43:04,534 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:43:04,538 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:43:04,541 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:43:14,710 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:43:23,021 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:43:28,104 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:43:31,378 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:43:31,385 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 4
2021-07-05 00:43:33,415 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 5
2021-07-05 00:43:38,876 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:43:39,591 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:43:39,595 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe4\x0c\xa8\x96\x81jd\x90/\xe0' RpcHandler Destroy!
2021-07-05 00:43:39,598 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:43:39,602 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e53ba896816a64902fe1 mem_id=2331085704352
2021-07-05 00:43:39,608 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e53ba896816a64902fe1 
2021-07-05 00:43:40,608 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:44:26,672 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:28,204 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:29,667 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:30,742 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:32,684 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:32,697 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:35,699 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:35,712 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:38,706 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:38,716 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:41,713 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:41,726 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:44,727 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:44,738 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:47,730 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:44:47,750 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:35,114 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:46:37,194 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:46:37,811 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:46:38,123 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:46:38,126 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 00:46:39,865 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:46:40,175 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:46:40,470 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 2
2021-07-05 00:46:40,782 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 00:46:41,140 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:46:41,143 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:46:41,146 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:46:41,149 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:46:41,150 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:46:41,151 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:46:41,162 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe5\xeb\xaf\x92&e\xeb\x8env' RpcHandler Destroy!
2021-07-05 00:46:41,163 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:46:41,165 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e5f1af922665eb8e6e77 mem_id=2087611330912
2021-07-05 00:46:41,167 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e5f1af922665eb8e6e77 
2021-07-05 00:46:41,172 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:46:41,204 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:41,225 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:44,210 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:44,221 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:47,216 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:47,227 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:50,223 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:50,234 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:53,224 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:53,236 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:56,229 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:56,241 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:59,226 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:46:59,238 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:02,237 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:02,247 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:05,253 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:05,262 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:08,276 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:08,285 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:11,285 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:11,297 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:14,302 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:14,312 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:23,258 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:47:23,303 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:47:23,308 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:47:23,313 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:47:23,320 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:47:23,326 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe6\x167\x1f\xef\xdb\xec\xc0>\xe6' RpcHandler Destroy!
2021-07-05 00:47:23,328 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:47:23,331 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e61b371fefdbecc03ee7 mem_id=2891451170000
2021-07-05 00:47:23,338 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e61b371fefdbecc03ee7 
2021-07-05 00:47:23,349 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:47:23,384 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:23,408 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:26,397 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:26,410 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:29,409 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:29,422 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:32,414 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:32,427 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:35,433 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:35,444 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:38,442 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:38,453 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:41,457 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:41,469 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:44,475 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:44,486 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:47,497 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:47,508 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:47:50,504 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:49:07,198 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 00:49:09,265 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:49:09,872 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 00:49:10,199 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:49:10,206 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:49:10,215 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:49:10,219 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:49:10,222 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9100)
2021-07-05 00:51:38,029 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:51:38,034 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe6\x83\xcc\xfbR\xc1\x91\xd17\xb8' RpcHandler Destroy!
2021-07-05 00:51:38,037 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:51:38,042 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e71accfb52c191d137b9 mem_id=2266607725296
2021-07-05 00:51:38,046 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e71accfb52c191d137b9 
2021-07-05 00:51:38,056 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:51:38,097 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:38,120 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:41,102 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:41,113 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:44,107 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:44,120 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:47,118 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:47,131 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:50,133 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:50,146 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:53,146 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:53,159 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:56,154 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:56,168 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:59,175 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:51:59,186 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:52:02,185 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:52:02,201 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:52:05,201 - DEBUG - TcpConn - [ConnBase.py:233]: send_data_and_count: peer_addr=('127.0.0.1', 9202)
2021-07-05 00:54:08,012 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 00:54:10,099 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:10,705 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 00:54:11,003 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:54:11,010 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:54:12,747 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:13,039 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:13,351 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 2
2021-07-05 00:54:13,647 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:54:14,022 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:54:15,404 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:15,699 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:16,010 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 3
2021-07-05 00:54:16,306 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 2
2021-07-05 00:54:17,038 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:54:18,063 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:18,342 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:18,668 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 4
2021-07-05 00:54:18,949 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 3
2021-07-05 00:54:20,056 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:54:20,725 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:20,995 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:21,348 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 5
2021-07-05 00:54:21,600 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 4
2021-07-05 00:54:23,061 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:54:23,068 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:54:23,076 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:54:30,605 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:30,609 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:54:36,085 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:54:36,091 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 6
2021-07-05 00:54:36,094 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:65"]: try 6 times , still can't connect tcp remote addr: ('127.0.0.1', 9101)
2021-07-05 00:54:36,097 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 5
2021-07-05 00:54:36,106 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe7\xaf\x1eHCPX0m\xec' RpcHandler Destroy!
2021-07-05 00:54:36,108 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:54:36,111 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e7cc1e48435058306ded mem_id=1407449652240
2021-07-05 00:54:36,114 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e7cc1e48435058306ded 
2021-07-05 00:54:36,124 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:56:58,716 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:57:12,072 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:57:24,106 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:57:24,112 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:57:24,119 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:57:41,950 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:57:41,954 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:58:12,426 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:58:14,507 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:58:15,113 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:58:15,442 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:58:15,445 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:58:15,449 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:58:15,460 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe8\xa4}<m\x14\x8dRp{' RpcHandler Destroy!
2021-07-05 00:58:15,461 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:58:15,463 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e8a77d3c6d148d52707c mem_id=2694185423152
2021-07-05 00:58:15,465 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e8a77d3c6d148d52707c 
2021-07-05 00:58:15,470 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:58:19,078 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:58:19,112 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:58:19,119 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe8\xab\x9aff\xfe\xf7AXv' RpcHandler Destroy!
2021-07-05 00:58:19,120 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:58:19,121 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e8ab9a6666fef7415877 mem_id=1938578142256
2021-07-05 00:58:19,123 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e8ab9a6666fef7415877 
2021-07-05 00:58:19,128 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:58:23,638 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:58:23,673 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:58:23,679 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe8\xaf\xe3\x07\x99\xbb\xc6\xedY\x01' RpcHandler Destroy!
2021-07-05 00:58:23,680 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:58:23,682 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e8afe30799bbc6ed5902 mem_id=2657410674992
2021-07-05 00:58:23,684 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e8afe30799bbc6ed5902 
2021-07-05 00:58:23,689 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:58:26,757 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:58:26,796 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:58:26,803 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe8\xb2*\x89\x07S\x91\x94\xa2\x99' RpcHandler Destroy!
2021-07-05 00:58:26,803 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:58:26,806 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e8b22a8907539194a29a mem_id=2061976465808
2021-07-05 00:58:26,807 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e8b22a8907539194a29a 
2021-07-05 00:58:26,811 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 00:58:29,959 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 00:58:32,025 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:58:32,630 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 00:58:32,973 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:58:32,975 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 00:58:34,670 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:58:35,028 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 00:58:35,278 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 2
2021-07-05 00:58:35,635 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 00:58:35,995 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 00:58:35,998 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 00:58:36,001 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 00:58:36,015 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xe8\xb5\xac\xder#p\xa6%\x18' RpcHandler Destroy!
2021-07-05 00:58:36,017 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_1', '127.0.0.1', 9202)
2021-07-05 00:58:36,019 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1e8bcacde722370a62519 mem_id=3131767132512
2021-07-05 00:58:36,021 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1e8bcacde722370a62519 
2021-07-05 00:58:36,026 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9202) is connected !!!!
2021-07-05 01:00:30,757 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:00:32,838 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:00:33,445 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 01:00:33,774 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:00:35,488 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:00:36,095 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 2
2021-07-05 01:00:36,783 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:00:38,153 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:00:38,757 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 3
2021-07-05 01:00:39,797 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:00:40,794 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:00:41,398 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 4
2021-07-05 01:00:42,819 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:00:43,440 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:00:44,045 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 5
2021-07-05 01:00:45,835 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:00:46,098 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:00:46,705 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 6
2021-07-05 01:00:46,706 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:65"]: try 6 times , still can't connect tcp remote addr: ('127.0.0.1', 9101)
2021-07-05 01:00:48,840 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:00:48,843 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:00:51,835 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:00:51,838 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:00:54,854 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:00:54,857 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:00:57,871 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:00:57,874 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:00,872 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:00,876 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:03,877 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:03,881 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:06,897 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:06,900 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:09,916 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:09,919 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:12,937 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:12,939 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:15,956 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:15,959 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:18,971 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:18,973 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:21,989 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:21,992 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:25,002 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:25,006 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:28,020 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:28,023 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:31,035 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:31,039 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:34,056 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:34,059 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:37,076 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:37,079 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:40,089 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:40,092 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:43,082 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:43,086 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:46,094 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:46,097 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:49,105 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:49,108 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:52,100 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:52,103 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:55,116 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:55,119 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:01:58,136 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:01:58,139 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:02:01,140 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:02:01,142 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:02:04,158 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:02:04,163 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:05:27,350 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:05:29,415 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:05:30,022 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 01:05:30,365 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:05:32,078 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:05:32,682 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 2
2021-07-05 01:05:33,388 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:05:33,391 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 01:05:33,395 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 01:05:33,404 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xeaW\xb2?K\xa3\x18b"\x8a' RpcHandler Destroy!
2021-07-05 01:05:33,405 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_0', '127.0.0.1', 9201)
2021-07-05 01:05:33,407 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1ea5db23f4ba31862228b mem_id=1575603544368
2021-07-05 01:05:33,409 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1ea5db23f4ba31862228b 
2021-07-05 01:05:33,414 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9201) is connected !!!!
2021-07-05 01:07:42,568 - DEBUG - RpcHandler - [RpcHandler.py:217]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 01:07:44,645 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:07:45,255 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 01:07:45,582 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:07:45,585 - DEBUG - RpcHandler - [RpcHandler.py:217]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:07:47,293 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:07:47,620 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:07:47,900 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 2
2021-07-05 01:07:48,228 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 01:07:48,604 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:07:48,607 - DEBUG - RpcHandler - [RpcHandler.py:217]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 01:07:48,611 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 01:07:48,620 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xea\xde@\x96\x0eS)\x90\xffL' RpcHandler Destroy!
2021-07-05 01:07:48,621 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_0', '127.0.0.1', 9201)
2021-07-05 01:07:48,622 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1eae440960e532990ff4d mem_id=2783386673888
2021-07-05 01:07:48,624 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1eae440960e532990ff4d 
2021-07-05 01:07:48,636 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9201) is connected !!!!
2021-07-05 01:15:53,092 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:15:55,163 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:15:55,769 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 01:15:56,096 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:15:56,099 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 01:15:56,102 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 01:15:56,110 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:83"]: self.rpc_handler_id=b'`\xe1\xec\xc9@257\x04\xa7K\x87' RpcHandler Destroy!
2021-07-05 01:15:56,111 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_0', '127.0.0.1', 9201)
2021-07-05 01:15:56,113 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1eccc4032353704a74b88 mem_id=2073820131392
2021-07-05 01:15:56,115 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1eccc4032353704a74b88 
2021-07-05 01:15:56,118 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:239"]: 
2021-07-05 01:15:56,121 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9201) is connected !!!!
2021-07-05 01:20:10,179 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9102)
2021-07-05 01:20:12,267 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:20:12,873 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 1
2021-07-05 01:20:13,201 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:20:13,204 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9101)
2021-07-05 01:20:14,926 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:20:15,254 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:20:15,537 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 2
2021-07-05 01:20:15,863 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 1
2021-07-05 01:20:16,221 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:20:17,591 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:20:17,917 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:20:18,212 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 3
2021-07-05 01:20:18,524 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 2
2021-07-05 01:20:19,242 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:20:20,270 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:20:20,582 - ERROR - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:54"]: [WinError 1225] The remote computer refused the network connection
2021-07-05 01:20:20,879 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9102) ... 4
2021-07-05 01:20:21,191 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:57"]: try reconnect tcp: ('127.0.0.1', 9101) ... 3
2021-07-05 01:20:22,253 - ERROR - __main__ - ["C:/Users/b/Documents/github/realtime-server/pycharm2020.1.3/script/client/TcpClient2.py:73"]: _err="request rpc timeout, rpc_fuc_name='pick_lowest_load_service_addr', rpc_fuc_args=['lobby_gate'], rpc_fuc_kwargs={}"
2021-07-05 01:20:22,256 - DEBUG - RpcHandler - [RpcHandler.py:218]: _handle_create_conn: addr=('127.0.0.1', 9100)
2021-07-05 01:20:22,272 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9100) is connected !!!!
2021-07-05 01:20:22,283 - DEBUG - __main__ - [TcpClient2.py:84]: lobby server info: ('gate_0', '127.0.0.1', 9201)
2021-07-05 01:20:22,285 - INFO - ServerEntity.Avatar - [ServerEntity.py:25]: __init__ create entity Avatar with id 60e1edd64ad9ca5629816694 mem_id=1882387063616
2021-07-05 01:20:22,286 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60e1edd64ad9ca5629816694 
2021-07-05 01:20:22,290 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:239"]: cancel _handle_create_conn task addr=('127.0.0.1', 9102)
2021-07-05 01:20:22,292 - WARNING - RpcHandler - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\RpcHandler.py:239"]: cancel _handle_create_conn task addr=('127.0.0.1', 9101)
2021-07-05 01:20:22,294 - INFO - TcpProtocol - [ConnMgr.py:53]: TCP ROLE_TYPE_ACTIVE peer_addr=('127.0.0.1', 9201) is connected !!!!
